test/test_control_client.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
```

Comment:
These tests shouldn't need the shebang either (they're never executed directly, right?)

Same for the test_control_server.py file.